src/app/examples/segmented-control-example/segmented-control-example.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit } from '@angular/core';
+
+import { SegmentItem } from '~/kirby/components/segmented-control/segment-item';
+
+@Component({
+  selector: 'kirby-segmented-control-example',
+  templateUrl: './segmented-control-example.component.html',
+  styleUrls: ['./segmented-control-example.component.scss'],
+})
+export class SegmentedControlExampleComponent implements OnInit {
+  activeSegment: string;
```

Comment:
Just use the item itself:
```suggestion
  activeSegment: SegmentItem;
```